lib/registries/addon/overview/index/template.hbs

```diff
@@ -1,4 +1,4 @@
-{{#unless this.registration.isRoot}}
+{{#if this.registration.isRoot}}
```

Comment:
If the `registration` itself is already the root registration, it is not a component of any parent registration.

Reply:
Got it, I have updated the structure of the project tested to not be registered before any components are.